src/CosmosDB/CosmosDB/help/Get-AzCosmosDBMongoDBRestorableResource.md

```diff
@@ -0,0 +1,153 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll-Help.xml
+Module Name: Az.CosmosDB
+online version: https://docs.microsoft.com/en-us/powershell/module/az.cosmosdb/get-azcosmosdbmongodbrestorableresource
+schema: 2.0.0
+---
+
+# Get-AzCosmosDBMongoDBRestorableResource
+
+## SYNOPSIS
+Lists all the restorable Azure Cosmos DB MongoDB resources available for a specific database account at a given time and location.
```

Comment:
can you call out the difference between Get-AzCosmosDBMongoDBRestorableResource and Get-AzCosmosDBMongoDBRestorableDatabase/Get-AzCosmosDBMongoDBRestorableCollection. How is this one different from the previous two, in which scenarios user should call this versus the other two.

Reply:
Added the detailed descriptions to describe when each of the command should be used.